flow360/component/simulation/meshing_param/volume_params.py

```diff
@@ -41,25 +56,99 @@ class AxisymmetricRefinement(Flow360BaseModel):
     )
     entities: EntityList[Cylinder] = pd.Field()
     # pylint: disable=no-member
-    spacing_axial: LengthType.Positive = pd.Field()
-    spacing_radial: LengthType.Positive = pd.Field()
-    spacing_circumferential: LengthType.Positive = pd.Field()
 
 
-class RotationCylinder(AxisymmetricRefinement):
-    """This is the original SlidingInterface. This will create new volume zones
+VolumeRefinementTypes = Union[UniformRefinement, AxisymmetricRefinement]
+
+
+class RotationCylinder(CylindricalRefinementBase):
+    """
+    This is the original SlidingInterface. This will create new volume zones
     Will add RotationSphere class in the future.
     Please refer to
     https://www.notion.so/flexcompute/Python-model-design-document-
     78d442233fa944e6af8eed4de9541bb1?pvs=4#c2de0b822b844a12aa2c00349d1f68a3
+
+    - `enclosed_entities` is actually just a way of specifying the enclosing patches of a volume zone.
+    Therefore in the future when supporting arbitrary-axisymmetric shaped sliding interface, we may not need this
+    attribute at all. For example if the new class already has an entry to list all the enclosing patches.
+
     """
 
-    name: Optional[str] = pd.Field(None)
```

Comment:
can we keep "name" in this PR and update all names all together when we have decision?

Reply:
Ok